app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,97 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  SummaryBox,
+} from '@trussworks/react-uswds'
+import { GitHubLabel } from '~/components/GitHubLabel'
+
+# Setting up New GCN Notice Producers
+
+The following steps guide new instrument, mission, or observatory producers
+into setting up new notices streams that are distributed to the user community
+via [Kafka](faq#what-is-kafka). This process requires interactions
+with the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+to enable these accounts and topics are setup on the GCN Kafka broker,
+and we are happy to work together to help you define schema utilizing our
+core JSON schema.
```

Comment:
This sentence is long and confusing. It has too many long clauses joined by "and", some of which are independent clauses and some of which are dependent classes. Split it into two or more sentences.